assignments/bootstrap/resume/resume.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Resume</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style_resume.css">
+    <link rel="stylesheet" href="bootstrap.min.css">
+</head>
+
+<body>
+    <div class="container-fluid">
```

Comment:
container rather than -fluid is better in this case as it will center your content rather than stretching it.